src/Courses/CourseDetailsTabs/StatisticsTab.tsx

```diff
@@ -162,83 +227,220 @@ export const StatisticsTab: React.FC<StatisticsTabProps> = ({course, userId}) =>
             console.log(userId);
             if (userId !== undefined) {
                 setIdFilter(rowData.id);
                 console.log('Switching to Attempts');
-                setView(StatisticsView.ATTEMPTS);
+                resetBreadCrumbs(StatisticsView.PROBLEMS, newBreadcrumb);
+                setView(StatisticsViewFilter.PROBLEMS_FILTERED);
             } else {
                 console.log('Showing a panel.');
                 togglePanel();
             }
             break;
         case StatisticsView.ATTEMPTS:
+        case StatisticsViewFilter.PROBLEMS_FILTERED:
             togglePanel();
             break;
         default:
             break;
-        }   
+        }
     };
-    
-    const hasDetailPanel = userId !== undefined ? 
-        view === StatisticsView.ATTEMPTS :
-        view === StatisticsView.PROBLEMS;
-    
+
+    const getTitle = (): string => {
+        let result = course.name;
+        Object.keys(StatisticsView).forEach((key: string) => {
+            result = breadcrumbFilter[key as StatisticsView]?.displayName ?? result;
+        });
+        return result;
+    };
+
+    const hasDetailPanel = userId !== undefined ?
+        (view === StatisticsView.ATTEMPTS || view === StatisticsViewFilter.PROBLEMS_FILTERED):
+        (view === StatisticsView.PROBLEMS || view === StatisticsViewFilter.TOPICS_FILTERED);
+
     let seeMoreActions: Array<any> | undefined = hasDetailPanel ? undefined : [{
-        icon: () => <ChevronRight/>,
+        icon: () => <ChevronRight />,
         tooltip: 'See More',
-        onClick: _.curryRight(nextView)(()=>{}),
+        onClick: _.curryRight(nextView)(() => { }),
     }];
-
     return (
         <>
             <Nav fill variant='pills' activeKey={view} onSelect={(selectedKey: string) => {
-                setView(selectedKey);
+                setView(selectedKey as StatisticsViewAll);
+                setBreadcrumbFilters({});
                 setIdFilter(null);
             }}>
-                <Nav.Item>
-                    <Nav.Link eventKey={StatisticsView.UNITS}>
-                        Units
-                    </Nav.Link>
-                </Nav.Item>
-                <Nav.Item>
-                    <Nav.Link eventKey={StatisticsView.TOPICS}>
-                        Topics
-                    </Nav.Link>
-                </Nav.Item>
-                <Nav.Item>
-                    <Nav.Link eventKey={StatisticsView.PROBLEMS}>
-                        Problems
-                    </Nav.Link>
-                </Nav.Item>
-                {userId !== undefined && <Nav.Item>
-                    <Nav.Link eventKey={StatisticsView.ATTEMPTS}>
-                        Attempts
-                    </Nav.Link>
-                </Nav.Item>}
+                {Object.keys(StatisticsView).map((key: string) => (
+                    (key !== StatisticsView.ATTEMPTS || userId !== undefined) &&
+                    <Col key={`global-${key}`} className="p-0" >
```

Comment:
What is this classname?

Reply:
padding 0, instead of doing inline styles we should probably use bootstrap classes more, even if we move away from react bootstrap